pages/linux/ufw.md

```diff
@@ -11,26 +11,26 @@
 
 `ufw disable`
 
-- Show ufw rules along with their numbers:
+- Show ufw rules, along with their numbers:
 
 `ufw status numbered`
 
 - Allow incoming traffic on port 5432 on this host:
 
 `ufw allow {{5432}}`
 
-- Allow traffic from 192.168.0.4 to any address on this host on port 22 using only tcp protocol:
+- Allow traffic from 192.168.0.4 to any address on this host, on port 22, using only the tcp protocol:
 
 `ufw allow from {{192.168.0.4}} to {{any}} port {{22}} proto {{tcp}}`
 
 - Deny traffic on port 80 on this host:
 
 `ufw deny {{80}}`
 
-- Deny traffic to port 22 using only udp protocol:
+- Deny traffic to port 22, using only the UDP protocol:
```

Comment:
Hmm. I beg to disagree. My arguments being:

1. Less text is faster to scan/read/maintain (and also store and transfer)
2. The point of the example is that the UDP traffic will be blocked on port 22, emphasis on the kind of traffic and not the port

Perhaps we can do an example like:

```markdown

- Deny all [UDP,TCP] traffic to port 22:

`ufw deny from {{any}} to {{any}} port {{22}} proto {{udp|tcp}}`
```

Otherwise I stand by my suggestion, but I'd like to hear @waldyrious's opinion on it